src/Features/CSharp/Portable/UsePatternMatching/CSharpAsAndNullCheckCodeFixProvider.cs

```diff
@@ -45,18 +45,31 @@ public override Task RegisterCodeFixesAsync(CodeFixContext context)
             return SpecializedTasks.EmptyTask;
         }
 
-        private void AddEdits(
+        private static ExpressionSyntax GetCondition(SyntaxNode node)
+        {
+            switch (node.Kind())
+            {
+                case SyntaxKind.WhileStatement:
+                    return ((WhileStatementSyntax)node).Condition;
+                case SyntaxKind.IfStatement:
+                    return ((IfStatementSyntax)node).Condition;
+                default:
+                    return null;
```

Comment:
It can't, just to satisfy the compiler. is there an ExceptionUtilities.Unreachable accessible to use?